src/convertToObject.js

```diff
@@ -3,16 +3,38 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ // eslint-disable-next-line max-len
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const normalizedString = sourceString
+    .replace(',', '')
+    .replace('  ', '');
+
+  const complexProperties = normalizedString
+    .split(';')
+    .map(element => {
+      const returnedElement = element.split(':');
+
+      return returnedElement;
```

Comment:
here you can write shorter solution ex: 
```suggestion
    .map(element => element.split(':'));
```